sandboxed_containers/sandboxed-containers-release-notes.adoc

```diff
@@ -0,0 +1,167 @@
+
+[id="sandboxed-containers-1-2-bug-fixes"]
+== Bug fixes
+
+* Previously, using loop devices in a sandboxed container was not possible due to missing kernel modules. With this release, these kernel modules are included in the package. (link:https://issues.redhat.com/browse/KATA-1334[*KATA-1334*])
+
+* Previously, the `MachineConfigPool` (MCP) created by the Operator to track nodes with the `Kata` runtime installed was not automatically removed on deletion of the `KataConfig` custom resource (CR). With this release, the deletion of the `KataConfig` CR results in the removal of the `kata-oc` MCP. (link:https://issues.redhat.com/browse/KATA-1184[*KATA-1184*])
+
+* Previously, when you created a `kataConfigPoolSelector` field and changed it, the {sandboxed-containers-operator} did not apply the change. With this release, the Operator acts on changes to the `kataConfigPoolSelector` field in the CRD and adapts installations of the runtime on nodes accordingly. (link:https://issues.redhat.com/browse/KATA-1190[*KATA-1190*])
+
+* Previously, the `SourceImage` field was displayed on the web console and using the field had no effect on the installation. With this release, the unused `SourceImage` field is no longer displayed when creating the `KataConfig` custom resource from the web console. (link:https://issues.redhat.com/browse/KATA-1015[*KATA-1015*])
+
+[id="sandboxed-containers-1-2-known-issues"]
+== Known issues
+
+* If you are using {sandboxed-containers-first}, you might receive SELinux denials accessing files or directories mounted from the `hostPath` volume in an {product-title} cluster. These denials can occur even when running privileged sandboxed containers, since privileged sandboxed containers do not disable SELinux checks.
++
+Following SELinux policy on the host guarantees full isolation of the host file system from the sandboxed workload by default, and provides stronger protection against potential security flaws in `virtiofsd` or QEMU.
++
+If the mounted files or directories do not have specific SELinux requirements on the host, you can use local persistent volumes as an alternative. Files are automatically relabeled to `container_file_t`, following SELinux policy for container runtimes. See xref:../storage/persistent_storage/persistent-storage-local.adoc[Persistent storage using local volumes] for more information.
++
+Automatic relabeling is not an option when mounted files or directories are expected to have specific SELinux labels on the host. Instead, you can set custom SELinux rules on the host in order to allow virtiofsd to access these specific labels. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1904609[*BZ#1904609*])
+
+* You might encounter an issue where `machine-config-operator` (MCO) pod changes to `CrashLoopBackOff` state and the `openshift.io/scc` annotation of the POD shows "sandboxed-containers-operator-scc" instead of the default `hostmount-anyuid`.
```

Comment:
```suggestion
* You might encounter an issue where the Machine Config Operator (MCO) pod changes to a `CrashLoopBackOff` state and the `openshift.io/scc` annotation of the pod displays `sandboxed-containers-operator-scc` instead of the default `hostmount-anyuid` value.
```